spec/src/main/asciidoc/chapters/introduction/entities.asciidoc

```diff
@@ -0,0 +1,142 @@
+== Type system
+
+This specification assumes that data is represented in a structured form, that is, that it is expressed in terms of:
+
+1. **atomic values**,
+2. **collections**, and
+3. **structures**.
+
+Any atomic value, collection, or structure belongs to at least one well-defined type.
+
+=== Atomic values
+
+_Atomic values_ are things defined externally to this specification, taken to include--at bare minimum--strings of Unicode characters, integers, floating point numbers, and dates, times, and datetimes.
```

Comment:
Consider if character and boolean should be included here as well:
```suggestion
_Atomic values_ are things defined externally to this specification, taken to include--at bare minimum--strings of Unicode characters, individual characters, booleans, integers, floating point numbers, and dates, times, and datetimes.
```

Reply:
Boolean definitely. I'm thinking to exclude from this list `char`, since some PLs don't even have a distinct character type.

Also note that we'll probably eventually have a lot more to say about these basic types. I just don't want to get into details at this point.